acceptance/analyzer_test.go

```diff
@@ -158,23 +158,31 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 
-	when("layers path and analyzed path are provided", func() {
-		it("writes analyzed.toml at the provided path", func() {
+	when("analyzed path is provided", func() {
+		it("writes analyzed.toml at the provided path and uses group.toml from the working directory", func() {
+			// The working directory is set to /layers in the Dockerfile
+
+			cacheVolume = h.SeedDockerVolume(t, cacheFixtureDir)
+
 			h.DockerRunAndCopy(t,
 				containerName,
 				copyDir,
 				analyzeImage,
-				"/other-layers/other-analyzed.toml",
-				h.WithFlags("--env", "CNB_REGISTRY_AUTH={}"),
+				"/",
```

Comment:
Copying the entire filesystem out seems like overkill potentially. Seems like we potentially need to refactor the helpers here to accept multiple files to copy out.

Reply:
Following https://github.com/buildpacks/lifecycle/pull/454#discussion_r520737124, we don't need to change `DockerRunAndCopy`.